README.md

```diff
@@ -8,6 +8,48 @@ How to use:
 * Start server: go run main.go server
 * Client call to check health status of server: go run main.go client.
 
+### Instruction for Windows Service Installation
+
+If you want to install harness-docker-runner as a service in windows please follow the bellow instructions.
+
+#### Instruction from creating msi from Exe file.
+* Here is official reference document: https://github.com/mh-cbon/go-msi
+* install go-msi package
+* create the binary for harness-docker-runner-svc
+```
+  go build -o lite-engine.exe
+```
+* run the below command from root directory to create msi
+```
+  go-msi make --msi harness-docker-runner-svc.msi --version your_version
+```
+
+#### Installation
+* Download the msi (harness-docker-runner-svc.msi) from latest releases
+* Double click the msi to start the installation process (If you see any error while opening the file please follow the instruction from below url)
+* https://answers.microsoft.com/en-us/windows/forum/all/this-installation-package-could-not-be/d6d913e9-aac7-429a-ac0d-c39ad3a7c5eb
+* Accept the liecence, click next and finish the installation.
+#### Uninstallation
+* Search and open add or remove programs from search menu
+* search for harness-docker-runner-svc
+* click Uninstall and finish.
+#### Additional Instructions
+* Service will automatically started even if you re-started your VM
+* You can manually Stop/Start the service from service menu tab.
+* Please do not delete the service manually from the services it may cause issue in uninstallation.
+* In case you have manually deleted the service follow below step to start service again:
+* Download the binary from release for windows
+* open the cmd as administrator and run below command
+```
+=== start the service ===
+sc.exe create harness-docker-runner-svc binPath="Path/to/your/binary server"
```

Comment:
make sure harness-docker-runner-svc doesn't exist before we do this. 
binPath="Path/to/your/harness-docker-runner.exe server"

Reply:
sc.exe start harness-docker-runner-svc